grakn-test/pom.xml

```diff
@@ -29,16 +29,15 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>grakn-test</artifactId>
-    <packaging>jar</packaging>
+    <packaging>pom</packaging>
+    <modules>
+        <module>test-unit</module>
```

Comment:
Maybe `test-behaviours` and `test-snb` ? The tests aren't really unit tests anyway?